share/scripts/inverse/csg_table

```diff
@@ -96,6 +96,10 @@ postadd overwrite postadd_overwrite.sh
 postadd plot postadd_plot.sh
 postadd average postadd_average.sh
 
+# smoothing for updated potential to allow forgetting noise from previous iterations (rdfs)
+# the script is the same as postupd_smooth.sh
+postadd smooth postadd_smooth.sh
```

Comment:
I did not mean to substitute smoothing the dU updates! This is just an extra option which I use at the beginning and intermediate stages of IBI or IMC: 2 or 3 smoothing loops for dU and 1 for U_new. I noticed that otherwise some strong noise features (due to poorly populated bins/regions) take very long to get smoothed (or can prevail unless I remove them manually).